### PR TITLE
Quick Fixes to README

### PR DESCRIPTION
**Pull request by DOUARA**
quick fixes but I believe they worth it: 

- Added WooCommerce to the list of required plugins 
- Added the file `wooConfig.example.js` which was missing

### CODE REVIEW

File: wooConfig.example.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const wooConfig = {
+	siteUrl: 'https://xyz.com',
+	graphqlUrl: 'https://xxwordpresssitexx.com/graphql',
```
Comment (imranhsayed) [inline wooConfig.example.js:3]: We should move the config into a `.env` file
https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

**Review by imranhsayed — CHANGES_REQUESTED** [review]

File: wooConfig.example.js (hunk 1 above)
Comment (DOUARA) [reply wooConfig.example.js:3]: @imranhsayed We already have `.env` file with the following content:
```
SITE_URL=http://localhost:3000
WOO_SITE_URL=http://yourwocommercesite.com
WOO_CONSUMER_KEY=xxxxx
WOO_SECRET=xxxxx

```

in that regard, may be we don't need this file `wooConfig.example.js`

Comment (imranhsayed) [reply wooConfig.example.js:3]: A few env variables are need server side while others client side. 
The one's we need client side must be prefixed by `NEXT_PUBLIC_`
e.g. NEXT_PUBLIC_WORDPRESS_SITE_URL=abcdefghijk
One's prefixed will be available both server side and client side.
We need WordPress site url for both client side and server side

We can remove :
```shell
WOO_CONSUMER_KEY=xxxxx
WOO_SECRET=xxxxx
```

Once we replace all instances we can safely remove this file and put `.env-example` instead

**Review by imranhsayed — CHANGES_REQUESTED** [review]

**Comment by NicestRudeGuy** [comment]
```
SITE_URL=http://localhost:3000
NEXT_PUBLIC_WOO_SITE_URL=http://wpdecoupled.local
WOO_CONSUMER_KEY=xxxxx
WOO_SECRET=xxxxx
````

**Pull request closed without merging**